.pre-commit-config.yaml

```diff
@@ -0,0 +1,129 @@
+---
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.2.0
+    hooks:
+      # File validity checkers
+      - id: check-yaml
+      - id: check-toml
+
+      # Simple cleanliness things
+      - id: fix-byte-order-marker
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: mixed-line-ending
+      - id: check-added-large-files
+        args: [--maxkb=500]
```

Comment:
I believe the 500kB is the default value[0], so it's not necessary to explicitly specify here.

[0]https://github.com/pre-commit/pre-commit-hooks/blob/main/pre_commit_hooks/check_added_large_files.py#L68

Reply:
Changed it to a comment :)